CONTRIBUTING.md

```diff
@@ -81,7 +81,17 @@ The following requirements must be fulfilled for a Pull request to be mergable t
 * If the Pull request requires changes in structure of user files (e.g. project/synth or other xml files) or flash configuration.
     * It must ensure that files or flash configuration created with the official firmware or previous community releases stay compatible or are automatically upgraded (upward compatibility).
     * If possible files and flash configuration created with the changes of the Pull request can be used with older firmwares not having the change (downward compatibility). Older firmwares must not break if a newer configuration was stored on the device.
-* If the Pull request changes end user behavoir or introduces new features a new entry in the [CommunityFeatures.md](CommunityFeatures.md) file needs to be created in the preexisting style describing the feature and it's options as a small manual to users. This includes all runtime and compile time flags which shall be named in respective sections.
+* If the Pull request changes end user behavoir or introduces new features a new entry in the [CommunityFeatures.md](Documentation/CommunityFeatures.md) file needs to be created in the preexisting style describing the feature and it's options as a small manual to users. This includes all runtime and compile time flags which shall be named in respective sections.
+
+### UI Changes
+
+Changes to the existing UI should follow the following process
+
+* Ensure they meet the guidelines set out in [UX Principles](Documentation/ux_principles.md)
```

Comment:
We should align CamelCase.md with UPPERCASE.md and snake_case.md. I don't know which is best but three different doesn't seem right :D

Reply:
UPPERCASE.md is what github wants for the top level documents (CONTRIBUTING/CODEOWNERS/ETC). The codebase is generally using snake_case for file names so I'll change community features to match

Reply:
Hmm that may not be mandatory - https://github.com/orgs/community/discussions/22673